cmd/common/utils/consts.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2022 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+const (
+	DockerDefaultSocketPath     = "/run/docker.sock"
+	CrioDefaultSocketPath       = "/run/crio/crio.sock"
+	ContainerdDefaultSocketPath = "/run/containerd/containerd.sock"
+)
```

Comment:
Still not possible to get these values from the docker/containerd/crio packages?